gradle.properties

```diff
@@ -1,12 +1,18 @@
 autoServiceVersion=1.0.1
-bigtableVersion=2.5.0
-jacksonVersion=2.13.3
-javaPoetVersion=1.11.1
-hadoopCommonVersion=3.3.3
-hbaseClientVersion=2.4.13
+avroVersion=1.11.1
+bigtableVersion=2.9.0
+jacksonVersion=2.15.2
+javaPoetVersion=1.13.0
+jerseyJsonVersion=1.20
+jettisonVersion=1.5.4
+jettyServerVersion=9.4.51.v20230217
+jsonSmartVersion=2.4.11
+hadoopVersion=3.3.5
```

Comment:
This is the dependency we need to pay attention too. It was already bumped from 2.x to 3.x in 3.3.0.